tests/test_load.py

```diff
@@ -33,11 +33,16 @@ def assert_database_tmp_folder_is_deleted():
     assert len([d for d in dirs if d.endswith("~")]) == 0
 
 
+@pytest.fixture(scope="module", autouse=False)
+def storage_format():
```

Comment:
I am not getting the grasp on this fixture: 

when calling `db.save` , the tests always seem to use the `TableStorageFormat`  default which is  `csv`.